grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -1084,7 +1084,7 @@ class StockMovementService {
         shipment.shipmentNumber = identifierService.generateShipmentIdentifier()
         shipment.expectedShippingDate = new Date()
         shipment.name = order.name
-        shipment.description = order.description
+        shipment.description = order.orderNumber
```

Comment:
I think this should be taken form order.name not orderNumber, because description during creating PO is put into name not into orderNumber.

Reply:
@awalkowiak https://pihemr.atlassian.net/browse/OBPIH-2469 It is original ticket. You can read in there that is should be PO number.

Reply:
Ok, it is fine then